component/authenticator/pom.xml

```diff
@@ -50,8 +50,16 @@
                     <groupId>org.slf4j</groupId>
                     <artifactId>log4j-over-slf4j</artifactId>
                 </exclusion>
+                <exclusion>
+                    <groupId>org.wso2.carbon.identity</groupId>
+                    <artifactId>org.wso2.carbon.identity.application.authentication.framework</artifactId>
+                </exclusion>
             </exclusions>
         </dependency>
+        <dependency>
+            <groupId>org.wso2.carbon.identity.framework</groupId>
+            <artifactId>org.wso2.carbon.identity.application.authentication.framework</artifactId>
```

Comment:
where are we using this dependency?

Reply:
Currently dependency comes as a transitive dependency from `org.wso2.carbon.extension.identity.authenticator.utils` which is an older version (5.0.7)